src/lib.rs

```diff
@@ -159,7 +162,9 @@ pub trait JsErrorClass: std::error::Error + Send + Sync + 'static {
   /// Additional properties that should be defined on the error in JavaScript side.
   fn get_additional_properties(
     &self,
-  ) -> Option<Vec<(Cow<'static, str>, Cow<'static, str>)>>;
+  ) -> Vec<(Cow<'static, str>, Cow<'static, str>)>;
+
+  fn as_any(&self) -> &dyn Any;
```

Comment:
no default implementaiton since this retuns a trait object that references the implementer, so a defautl impl isnt allowed